### DISCUSSION TITLE
Emojis for Spaces and / or image upload for spaces (icon)

### DISCUSSION THREAD

**Discussion by proxforge** [docmost/docmost#146] [Ideas]
to make them easier to see and more beautiful:

![image](https://github.com/user-attachments/assets/131068bf-f222-432c-8073-a772e1976ac9)

**Comment by fcassata** [comment]
+1

Using the first two letters isn't really adding anything or making them easier to find at a glance. Both the workspace and spaces should have customizable icons or graphic. Preferably something one can upload, not just emojis.

**Reply by leadvic** [reply]
I agree, following the same path, I think groups should also have some avatars in them to easily identify them when you have too many

**Comment by WavedirectTel** [comment]
Add me to the list for this one.  Also to be able to upload our own emojis, including animated ones.

**Comment by vgromou** [comment]
+1

**Comment by kotiksmelyi** [comment]
+1

**Comment by TigranAvushyan** [comment]
+1

**Comment by EugeneKhavronin** [comment]
+1

**Comment by fh4dka** [comment]
+1

**Comment by I-Myeg** [comment]
+1

**Comment by haadi-coder** [comment]
+1

**Comment by fnordsh** [comment]
+1
